samples/boards/nucleo_l496zg/sensors/README.rst

```diff
@@ -0,0 +1,53 @@
+.. _sensor_samples:
+
+Sensor Samples
+##############
+
+Overview
+********
+
+This sample demonstrates the usage of the HDC1080 temp/rh sensor.
+
+Requirements
+************
+
+To use this sample, the following hardware is required:
+
+* Nucleo STM32L496ZG
```

Comment:
Is it? 
Won't it work with any board supporting zephyr with an I2C port?

Reply:
You are right, I reworked the README and now just mention this nucleo as a tested board.